src/js/view/view.js

```diff
@@ -340,9 +350,59 @@ function View(_api, _model) {
         playerViewModel.change('playlistItem', onPlaylistItem);
         // Triggering 'resize' resulting in player 'ready'
         _lastWidth = _lastHeight = null;
+
+        // Setup floating scroll handler
+        if (_floatingConfig && OS.mobile) {
+            _model.change('inDom', (model, val) => {
+                if (val) {
+                    viewsManager.addScrollHandler(throttledMobileFloatScrollHandler);
```

Comment:
So the player just got added to the DOM, but nothing happens until a scroll event...

Reply:
ahhh good call, will update this and fix the early exist in the scroll handler